test/avocado/seleniumlib.py

```diff
@@ -95,30 +95,38 @@ def tearDown(self):
             raise Exception('ERR: Unable to close WEBdriver', str(e))
 
     def everything_loaded(self, element):
+        """
+This function is only for internal purposes:
+    It via javascript check that attribute data-loaded is in element
+        """
         if javascript_operations:
-            for foo in range(0, self.default_try): 
-                if self.driver.execute_script("return Array.from(arguments[0].children).every(function (e) { return e.getAttribute('data-loaded'); })", element):
-                    break
+            return self.driver.execute_script("return arguments[0].getAttribute('data-loaded')", element)
+        else:
+            return True
 
     def click(self, element):
-        notdone=""
-        self.everything_loaded(element)
+        failure = "CLICK: too many tries"
+        usedfunction = self.element_wait_functions[element]
```

Comment:
This assumes that element_wait_functions contains the element, you should check for that to be the case.
I could wait for one element and just know that a second one will (should) be there by then.